e2e/pkg/testrunner/models.go

```diff
@@ -34,7 +34,7 @@ var _ = TestCaseSchema.Add("BeforeHookCustomSQL", `
 	"type": "object",
 	"additionalProperties": false,
 	"properties": {
-		"path": { "type": "string", "format": "file-path", "description": "Path to the custom SQL script" }
+		"path": { "type": "string", "format": "string", "description": "Path to the custom SQL script" }
```

Comment:
This is no `format: string` neither. Please remove "format".

Reply:
Removed 🙏